src/main/java/mate/controller/GetAllCarsController.java

```diff
@@ -0,0 +1,26 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+@WebServlet("/cars/all")
```

Comment:
```suggestion
@WebServlet("/cars")
```
Let's follow common mistakes advice. Please fix the same issues in all places